lib/generators/connector/templates/example_entity.rb

```diff
@@ -17,6 +17,10 @@
 #     ExampleEntityMapper
 #   end
 
+#   def self.creation_mapper_class
```

Comment:
Can you add a comment explaining that it is an optional workflow to be used only if needed?